README.md

```diff
@@ -1,26 +1,56 @@
 
-### Continous Integration Status
-|Rolling Build                    | Nightly Build                |
-|---------------------------------|:-----------------------------|
-|![Rolling-Build Status](https://devdiv.visualstudio.com/_apis/public/build/definitions/0bdbc590-a062-4c3f-b0f6-9383f67865ee/5974/badge) | ![Nightly-Build Status](https://devdiv.visualstudio.com/_apis/public/build/definitions/0bdbc590-a062-4c3f-b0f6-9383f67865ee/5976/badge) |
+## Continuous Integration Status
+
+| Rolling Build                                                                                                                           | Nightly Build                                                                                                                           |
+| --------------------------------------------------------------------------------------------------------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------- |
+| ![Rolling-Build Status](https://devdiv.visualstudio.com/_apis/public/build/definitions/0bdbc590-a062-4c3f-b0f6-9383f67865ee/5974/badge) | ![Nightly-Build Status](https://devdiv.visualstudio.com/_apis/public/build/definitions/0bdbc590-a062-4c3f-b0f6-9383f67865ee/5976/badge) |
 
 ## Get Started
+
 ### Prerequisite
+
 * [Application Insights for ASP.NET Core](https://github.com/Microsoft/ApplicationInsights-aspnetcore)
 * [Docker Containers](https://www.docker.com/)
 * [Kubernetes](https://kubernetes.io/)
 
-### Walkthrough
-We support **ASP.NET Core** application as well as **.NET Core** application.
+### Instrument an ASP.NET Core application
+
+These are the basic steps to instrument an ASP.NET Core application to enable Application Insights for Kubernetes. You will need to run the application in containers managed by Kubernetes to see the change.
+
+1. Add reference to **Application Insights SDK** and **Application Insights for Kubernetes**:
+
+    ```shell
+    dotnet add package Microsoft.ApplicationInsights.AspNetCore
+    dotnet add package Microsoft.ApplicationInsights.Kubernetes
+    ```
+
+2. Enable **Application Insights** and **Application Insights for Kubernetes Enricher** in `Startup.cs`:
+
+    ```csharp
+    public void ConfigureServices(IServiceCollection services)
+    {
+        ...
+        services.AddApplicationInsightsTelemetry();
+        services.AddApplicationInsightsKubernetesEnricher();
+        services.AddMvc();
+        ...
+    }
+    ```
+
+3. Build and run the app in containers managed by Kubernetes.

```

Comment:
Is it the case that the example will not work if Application Insights instrumentation key is not set? If so, I would mention it, as it is a common mistake.

Reply:
Updated.